todo.py

```diff
@@ -1,11 +1,39 @@
 import json
+import os
+from datetime import datetime
+
+def get_last_modified_timestamp(file_path):
+    try:
+        timestamp = os.path.getmtime(file_path)
+        return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
+    except FileNotFoundError:
+        return None
+    
+def load_tasks():
+    file_path = "tasks.json"
+    try:
+        with open(file_path, "r") as file:
+            tasks = json.load(file)
```

Comment:
you should handle the json error if the file is not read correctly
```suggestion
    try:
        with open(file_path, "r") as file:
            tasks = json.load(file)
    except json.JSONDecodeError:
        print("Error reading JSON from the file.")
```